bftengine/src/bcstatetransfer/BCStateTran.cpp

```diff
@@ -746,25 +761,31 @@ std::string BCStateTran::convertMillisecToReadableStr(uint64_t ms) const {
   auto days = n;
 
   std::string str;
+  bool shouldPad = false;
   if (days) {
-    str += std::to_string(days) + ":";
-    legend += "D:";
+    str += padZeroes(std::to_string(days), 2, shouldPad) + ":";
+    legend += "DD:";
```

Comment:
Should legend & shouldPad flag be passed & processed in existing/new lambda as well?

Reply:
That's a good idea, I'll make the changes. Thanks!

Reply:
shouldPad doesnt need ot be passed, just captured.